Automattic/jetpack
# External Media: Update Google Photo Filters

Fixes #15880

#### Changes proposed in this Pull Request:

* This addresses bugs discovered by dialing down the UI for filtering and starting with a few restricted options

#### Screenshot

<img width="829" alt="Screenshot 2020-06-18 at 18 15 56" src="https://user-images.githubusercontent.com/156676/85045906-f1007480-b18f-11ea-8b1a-27346c031316.png">

New filtering options allow you to:
- switch between All Photos (previously Recent Photos) and Albums
- for All Photos - we show the date range select with predefined options
- for Albums - we show breadcrumb links as you navigate in albums

In a separate PR, I will add an additional date range option that will allow you to customize it. Based on #15880 design

#### Testing instructions:
* Open the Google Photos modal
* Observe and test the new UI in the header of the modal - make sure it works as expected

#### Proposed changelog entry for your changes:
* none
